spec/helpers/ballots_helper_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe BallotsHelper do
+
+  describe "#remaining_votes" do
```

Comment:
Thank you for using helper specs, they are great and super fast to run 👍

No strong preference, but I think you could use `let!(:line)`, that should avoid the need to run the `before` block, as it ejecutes the initialization right away instead of waiting to the variable to be called 😉